README.md

```diff
@@ -52,9 +52,6 @@ Use real values instead of "var.<var_name>" or other placeholder values
 unless real values don't help users know what to change.
 -->
 
-#### Before You Begin
-If you are using this module to create an ATracker route and COS instance & bucket you will need an IAM authorization policy in the account where the KMS key resides which grants the Cloud Object Storage service in the account to which this solution is being deployed, reader access to the KMS instance that the KMS key belongs to.
```

Comment:
I think we still want to mention about the cross account use case. User needs to create policy in the account where the KMS key resides, and they need to set `skip_iam_authorization_policy = true`